### ISSUE TITLE
GBA BIOS is not always loaded (maybe not a problem of the emulator)

### ISSUE THREAD

**Issue by RunisDauphin** [mgba-emu/mgba#1447]
Hi, I am Runis Dauphin from Rebelles Unis:

I have just tried Megaman Battle Network 3 (both Blue and White) (Europe) with the BN3 Plus patch by Prof. 9, who restores the animations that were suppressed when bringing the game from Japan to the other regions.

I know that the patch is thought for the US version, but when soft patched to the European version it works everything except the BIOS. I do not know which think is not working, if it is a problem with the emulator or an incompatibility of the patch.

This is the patch I used:
http://www.romhacking.net/hacks/933/

Hope you can solve the problem if it is fixable.

**Comment by endrift** [comment]
Can you explain what you mean by "not working"? I don't understand the issue.

**Comment by RunisDauphin** [comment]
This was my test:

First, I put the patches to be loaded alongside the game (Megaman Battle Network 3 - Blue / White (Europe)), each of them with their respective patches. Second, I loaded the games (both of them, not simultaneously). Third, I realised when loading that the BIOS (configured and working with other games) was skipped. Fourth, I checked if the changes of the patch were being applied to see if it worked (and it did).

After that, I checked if the Megaman Battle Network 4 - Red Sun / Blue Moon Bug fix patch by Exeguy11 (http://www.romhacking.net/hacks/2264/), thought for the US version too, worked with the European releases. I loaded these two patches with their respective ROMs and they loaded, first the BIOS and then the game. I could not check if the Woodman stage was fixed because it is way too far in the story and sure would have me taken 2-3 months to get there.

What I was meaning about "not working" is that the BIOS, although configured and working with practically all the rest of the collection (as far as I know), was skipped in this game. That is what I mean.

Hope this gets a better understanding of the situation.

Runis Dauphin

P.S.: I do not give more information about the ROMs for now because I do not know how to check the checksum and I do not want to get flagged again for giving too much information.
